core/templates/pages/contributor-dashboard-page/modal-templates/translation-modal.component.spec.ts

```diff
@@ -131,6 +141,110 @@ describe('Translation Modal Component', () => {
     expect(changeDetectorRef.detectChanges).toHaveBeenCalledTimes(0);
   });
 
+  it('should expand the content area', () => {
+    spyOn(component, 'toggleExpansionState').and.callThrough();
+
+    // The content area is contracted by default.
+    expect(component.isContentExpanded).toBeFalse();
+
+    // The content area should expand when the users clicks
+    // on the 'View More' button.
+    component.toggleExpansionState('content');
+
+    expect(component.isContentExpanded).toBeTrue();
+  });
+
+  it('should contract the content area', () => {
+    spyOn(component, 'toggleExpansionState').and.callThrough();
+
```

Comment:
Nit: Can remove this new line since the below line is part of the test setup. Ditto below.

Reply:
Done